scripts/ci.sh

```diff
@@ -9,8 +9,16 @@ git clone https://github.com/hobinjk-ptc/test-spatialToolbox spatialToolbox
 
 cd addons
 
+git clone https://github.com/ptcrealitylab/pop-up-onboarding-addon
```

Comment:
I'm not sure exactly when this script gets executed, but just to double check – is it ok to point to this private addon repo in the open source server?

Reply:
A good thing to double check! Any pull request made from an external repository won't have access to the secrets required to pull any of the private repositories. Additionally, any first-time contribution won't run the tests at all without our approval.